examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,231 @@ Documentation for other releases can be found at
+the pod name from the `get pods` command, and then:
 
-Create the pod:
-
-```sh
-$ kubectl create -f examples/mysql-wordpress-pd/wordpress.yaml
+```shell
+kubectl logs <pod-name>
 ```
-
-And list the pods to check that the status of the new pod changes
-to `Running`.  As above, this might take a minute.
-
-```sh
-$ kubectl get pods
 ```
-
-### Start the WordPress service
-
-Once the wordpress pod is running, start its service, specified by [`wordpress-service.yaml`](wordpress-service.yaml).
-
-The service config file looks like this:
-
-<!-- BEGIN MUNGE: EXAMPLE wordpress-service.yaml -->
-
-```yaml
-apiVersion: v1
-kind: Service
-metadata: 
-  labels: 
-    name: wpfrontend
-  name: wpfrontend
-spec: 
-  ports:
-    # the port that this service should serve on
-    - port: 80
-  # label keys and values that must match in order to receive traffic for this service
-  selector: 
-    name: wordpress
-  type: LoadBalancer
+...
+2016-02-19 16:58:05 1 [Note] InnoDB: 128 rollback segment(s) are active.
+2016-02-19 16:58:05 1 [Note] InnoDB: Waiting for purge to start
+2016-02-19 16:58:05 1 [Note] InnoDB: 5.6.29 started; log sequence number 1626007
+2016-02-19 16:58:05 1 [Note] Server hostname (bind-address): '*'; port: 3306
+2016-02-19 16:58:05 1 [Note] IPv6 is available.
+2016-02-19 16:58:05 1 [Note]   - '::' resolves to '::';
+2016-02-19 16:58:05 1 [Note] Server socket created on IP: '::'.
+2016-02-19 16:58:05 1 [Warning] 'proxies_priv' entry '@ root@wordpress-mysql-cqcf4-9q8lo' ignored in --skip-name-resolve mode.
+2016-02-19 16:58:05 1 [Note] Event Scheduler: Loaded 0 events
+2016-02-19 16:58:05 1 [Note] mysqld: ready for connections.
+Version: '5.6.29'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server (GPL)
 ```
 
-[Download example](wordpress-service.yaml?raw=true)
-<!-- END MUNGE: EXAMPLE wordpress-service.yaml -->
+Also in [mysql-deployment.yaml](mysql-deployment.yaml) we created a
+service to allow other pods to reach this mysql instance. The name is
+`wordpress-mysql` which is resolvable to the pod IP.
 
-Note the `type: LoadBalancer` setting.  This will set up the wordpress service behind an external IP.
-Note also that we've set the service port to 80.  We'll return to that shortly.
+## Deploy WordPress
 
-Start the service:
+Next deploy WordPress using
+[wordpress-deployment.yaml](wordpress-deployment.yaml):
 
-```sh
-$ kubectl create -f examples/mysql-wordpress-pd/wordpress-service.yaml
+```shell
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/wordpress-deployment.yaml
 ```
 
-and see it in the list of services:
-
-```sh
-$ kubectl get services
-```
+Here we are using many of the same features, such as a volume claim
+for persistent storage and a secret for the password.
 
-Then, find the external IP for your WordPress service by running:
+The [WordPress image](https://hub.docker.com/_/wordpress/) can accept
```

Comment:
nit: a bit awkward.  Consider breaking up the sentence and dropping the conjunction.  Consider making the sentence more active + concise through s/can accept/accepts/ 

also s/databse/database/

The WordPress image accepts the database hostname through the environment variable `WORDPRESS_DB_HOST`.  We set the env value to the name of the MySQL service we created:  `wordpress-mysq`.